x-pack/plugins/ml/public/components/job_selector/_job_selector.scss

```diff
@@ -1,6 +1,5 @@
 .mlJobSelectorBar {
-  padding: 10px;
-  background-color: $euiColorLightShade
+  padding: $euiSizeS;
```

Comment:
At the moment the padding looks like this:

![image](https://user-images.githubusercontent.com/230104/57359569-476f4700-7178-11e9-9e15-314bcf22978d.png)

I'd like to suggest to align the padding with the elements further down (filter bar, Top Influencers) and change this to `$euiSize`, with this additional change in `_explorer.scss` (`0px` instead of `$euiSize` for top padding):
```css
  .results-container {
    padding: 0px 0px $euiSize 0px;
    ...
```

![image](https://user-images.githubusercontent.com/230104/57359360-d62f9400-7177-11e9-9236-2274cdb74d4c.png)